pkgs/by-name/gs/gsc/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchgit, cmake }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gsc";
+  version = "1.2";
```

Comment:
Are you sure about the version? https://git.launchpad.net/gsc/tree/debian/control?id=c6c29f0d6288ca42c32c6ead2f01e86662a2126a says 1.1.

But I'm not sure. At a glance, the code in the launchpad repo doesn't really match either the [1.1.](http://cdsarc.u-strasbg.fr/ftp/cats/bincats/GSC_1.1) or the [1.2](http://cdsarc.u-strasbg.fr/ftp/cats/bincats/GSC_1.2) version linked from the more authoritative [STScI website](https://gsss.stsci.edu/Catalogs/Catalogs.htm). Is there a reason to use the launchpad source and not the STSci package?

Reply:
Actually, it's version 1.3 according to the binary!

```
[bzizou@bart:~/git/cigri]$ gsc
How to search the Guide Star Catalogue (GSC-ACT or GSC1.3) by coordinates.
```